hi-ml/src/health_ml/model_trainer.py

```diff
@@ -91,9 +93,24 @@ def create_lightning_trainer(container: LightningContainer,
             message += "s per node with DDP"
     logging.info(f"Using {message}")
     tensorboard_logger = TensorBoardLogger(save_dir=str(container.logs_folder), name="Lightning", version="")
-    azureml_logger = AzureMLLogger(enable_logging_outside_azure_ml=container.log_from_vm,
-                                   run=azureml_run_for_logging)
-    loggers = [tensorboard_logger, azureml_logger]
+    loggers: List[Any] = [tensorboard_logger]
+
+    if is_running_in_azure_ml():
+        mlflow_run_id = os.environ.get("MLFLOW_RUN_ID", None)
+        mlflow_logger = MlflowLogger(
+            run_id=mlflow_run_id
+        )
+    else:
+        mlflow_run_dir = container.outputs_folder / "mlruns"
```

Comment:
so we have lost the ability to log to AzureML from a VM? (OK to punt that to a next PR)

Reply:
Yes I havent quite figured that out with mlfow.  It will be something like 
```
elif azureml_run_for_logging:
    tracking_uri = workspace.get_mlflow_tracking_uri()
    mlflow_logger = MlflowLogger(run_id=azureml_run_for_logging, tracking_uri=tracking_uri)
```
but that doesn't work so something is missing

Reply:
(Might need an `mlflow.start_run()`)